magento/adobe-stock-integration
# fixed correct Test Conection button validation result

### Description (*)

For some reason comparison by 'true === 1' in Javascript not correct and will return false. so we need to 'true === true' then this  will return true
### Fixed Issues (if relevant)

1. magento/adobe-stock-integration#271: Test Connection Result must be green When successful result
2. ...

### Manual testing scenarios (*)
1. Go to Store-> configuration -> system -> advanced -> Adobe Stock Config 
2. fill all required data push Test Connection
3. See green  Box
